src/generate/register.rs

```diff
@@ -788,7 +794,7 @@ pub fn fields(
                 }
             });
 
-            let description = &f.description;
+            let description = util::normalize_docstring(format!("{}", &f.description));
```

Comment:
Can these single parameter calls be simplified to `util::normalize_docstring(&f.description)`?

Reply:
no, I can't  (not without changing the signature of `util::normalize_docstring()` but it could bechanged to `util::normalize_docstring(f.description)`. I'm unsure if its ok to consume/move that string. `format!` though will yield/create a new string.

Reply:
FYI `util::normalize_docstring(f.description)` is compiling without complaint, so it seems nobody is accessing it after it was moved. Still it kinda feels wired to me (but I'm pretty new to rust).